fem/libceed/mass.hpp

```diff
@@ -12,21 +12,27 @@
 #ifndef MFEM_LIBCEED_MASS_HPP
 #define MFEM_LIBCEED_MASS_HPP
 
-#include "ceed.hpp"
-
+#include "pa_integrator.hpp"
 #include "../fespace.hpp"
 
 namespace mfem
 {
 
-/// Initialize a Mass Integrator using libCEED
-void CeedPAMassAssemble(const FiniteElementSpace &fes,
-                        const mfem::IntegrationRule &ir,
-                        CeedData& ceedData);
+class CeedPAMassIntegrator : public CeedPAIntegrator
+{
+public:
+   CeedPAMassIntegrator(const FiniteElementSpace &fes,
+                        const mfem::IntegrationRule &irm,
+                        Coefficient *coeff);
```

Comment:
`coeff` is called `Q` in the cpp file (and there is something different called `coeff` in the base class `CeedMFIntegrator`).